docs/releases.md

```diff
@@ -1,39 +1,157 @@
+
+<List of items with PRs and/or Issues to be considered for this release>
+
+# 🏗 API Changes
+
+<List of API changes, if any>
+
+# ✅ Release Checklist
+
+For each RC published in each stage:
 - [ ] version string in `version.go` has been updated
 - [ ] tag commit with vX.Y.Z-rcN
 
-## Pre-Release Checklist
-- [ ] before release, tag 'release candidate' for users to test against
-  - if bugs are found/fixed, do another release candidate
-- [ ] all tests pass (no exceptions)
-- [ ] run interop tests https://github.com/ipfs/interop#test-with-a-non-yet-released-version-of-go-ipfs
-- [ ] webui works (for most definitions of 'works') - Test the multiple pages and verify that no visible errors are shown.
+### Release Stage 1 - Internal testing
+
+When Release Stage 1, there is a features freeze for the release branch.
+
 - [ ] CHANGELOG.md has been updated
   - use `./bin/mkreleaselog` to generate a nice starter list
-- [ ] version string in `repo/version.go` has been updated
-- [ ] tag commit with vX.Y.Z
-- [ ] update release branch to point to release commit (`git merge vX.Y.Z`).
-- [ ] publish dist.ipfs.io
-- [ ] publish next version to https://github.com/ipfs/npm-go-ipfs
-
-## Post-Release
-- [ ] Bump version string in `repo/version.go` to `vX.Y.Z-dev`
-- [ ] Upload the final release to the github releases page: https://github.com/ipfs/go-ipfs/releases
-- Communication
-  - [ ] Create the release issue
-  - [ ] Announcements (both pre-release and post-release)
-    - [ ] Twitter
-    - [ ] IRC
-    - [ ] Reddit
-  - [ ] Blog post (at minimum, paste the changelog. optionally add context and thank contributors.)
-- [ ] Update HTTP-API Documentation on the Website using https://github.com/ipfs/http-api-docs
+- [ ] Automated Testing - Ensure that all tests are passing, this includes:
+  - [ ] unit
+  - [ ] sharness
+  - [ ] [interop](https://github.com/ipfs/interop#test-with-a-non-yet-released-version-of-go-ipfs)
+- [ ] Network Testing:
+  - [ ] test lab things
+- [ ] Infrastructure Testing:
+  - [ ] Deploy new version to a subset of Bootstrappers
+  - [ ] Deploy new version to a subset of Gateways
+  - [ ] Deploy new version to a subset of Preload nodes
+  - [ ] Collect metrics every day. Work with the Infrastructure team to learn of any hiccup
+- [ ] IPFS HTTP Client Libraries Testing:
+  - [ ] [JS](http://github.com/ipfs/js-ipfs-http-client)
+  - [ ] [Go](https://github.com/ipfs/go-ipfs-api)
+- [ ] IPFS Application Testing -  Run the tests of the following applications:
+  - [ ] WebUI
+  - [ ] IPFS Desktop
+  - [ ] IPFS Companion
+  - [ ] NPM on IPFS
+
+### Release Stage 2 - Invite IPFS partners to test
+
+- [ ] Reach out to the following IPFS partners for testing this release (check when no more problems have been reported):
+  - [ ] Infura
+  - [ ] Textila
+  - [ ] Open Bazaar
+  - [ ] QRI
+- [ ] Run tests available in the following repos with the latest RC (check when all tests pass):
+  - [ ] [orbit-db](https://github.com/orbitdb/orbit-db)
```

Comment:
I'm pretty sure that both of the lists here can be way bigger. We probably should have a "expectations description" before adding more folks

Reply:
Can we test this against go-ipfs? I thought it embedded a js-ipfs node.

Reply:
AFAIK, it can do both. Will check again

Reply:
(yes, sorry)